well-sensor-esp32.ino

```diff
@@ -21,6 +21,8 @@ void serialStr(char* str) {
 #include "sockets.h"
 #include "listen-ap.h"
 #include "init-mqtt.h"
+#include "data-median.h"
+#include "filter-median.h"
 
```

Comment:
Pls introduce init-data.h with data array global variable there. Include it before these lines